learn/what_is_meilisearch/comparison_to_alternatives.md

```diff
@@ -95,7 +95,7 @@ Can't find a client you'd like us to support? [Submit your idea or vote for it](
 |   | Meilisearch | Algolia | Typesense | Elasticsearch |
 |---|:---:|:----:|:---:|:---:|
 | Placeholder search | ✅ | ✅ | ✅ | ✅ |
-| Multi-index search | [2023](https://github.com/meilisearch/product/issues/74) | ✅ | ✅ | ✅ |
+| Multi-index search | ✅ | ✅ | ✅ | ✅ |
```

Comment:
```suggestion
| Multi-index search (Federated-search) | ✅ | ✅ | ✅ | ✅ |
```

Adding the term "Federated-search" that can talk to some expert visitors and start positioning us on that keyword